doc/source/point-in-time-recovery.rst

```diff
@@ -56,9 +56,15 @@ Run |pbm-restore| and specify the timestamp from the valid range:
 
 .. code-block:: bash
 
-   $ pbm restore --time="2020-07-14T14:27:04"
+   $ pbm restore --time="2020-12-14T14:27:04"
 
-Restoring to the point in time requires both a backup snapshot and oplog slices that can be replayed on top of it (the time ranges in PITR section of ``pbm list`` output must be later then the selected backup snapshot). 
+Restoring to the point in time requires both a backup snapshot and oplog slices that can be replayed on top of this backup. The time stamp you specify for the restore must meet the following conditions:
+
+* It must be from the time ranges in the PITR section of ``pbm list`` output
+* It must be later than the backup snapshot used for the restore. 
+
+To illustrate this behavior, let’s use the ``pbm list`` output from the previous example. For timestamp ``2020-07-14T14:27:04``, 
```

Comment:
Pls correct the date 2020-07-14 to 2020-12-14

Reply:
Thanks. Fixed.